File: week2/JavaScript/task4.js
function getNumber(index) {
  seq = [];
  for (let i=0; i < index; i++) {
    for (let j=0; j < 3; j++) {
        seq.push(i*7+j*4)
    }
  }
  console.log(seq[index])
}
console.log("==========Task 4==========");
getNumber(1); // print 4
getNumber(5); // print 15
getNumber(10); // print 25
getNumber(30); // print 70

File: week2/Python/task4.py
def get_number(index):
    seq=[]
    for i in range(index):
        for j in range(3):
            seq.append(i*7+j*4)
    print(seq[index])
print("==========TASK 4==========")
get_number(1) # print 4
get_number(5) # print 15 
get_number(10) # print 25 
get_number(30) # print 70